sml_builder/method.py

```diff
@@ -1,43 +1,72 @@
-from os import listdir
+from os import listdir, path
 from json import loads
 from flask import render_template
 from _jsonnet import evaluate_file  # pylint: disable=no-name-in-module
 from sml_builder import app
 from .utils import _page_not_found
 
-STATUS_CLASS = {
-    "In development": "pending",
-    "Complete": "success",
-    "Approved for development": "dead",
-}
-
 
 @app.route("/method/<method>")
```

Comment:
@jhmjam Am i good to remove this method entirely? My understanding is that this would be replaced by your new method below and i think the merge freaked out and kept both